ocs_ci/deployment/baremetal.py

```diff
@@ -199,9 +199,19 @@ def deploy_prereq(self):
             logger.info(rhcos_images_file)
             image_data = rhcos_images_file[ocp_version]
             # Download installer_initramfs
-            initramfs_image_path = (
-                constants.coreos_url_prefix + image_data["installer_initramfs_url"]
-            )
+
+            if Version.coerce(ocp_version) >= Version.coerce("4.12"):
```

Comment:
use version module  ( in other new places as well )

eg: version.get_semantic_ocp_version_from_config() >= version.VERSION_4_12

Reply:
Done